### PR TITLE
Win8 removed + bb10 & ubuntu depreciation info

### PR DESCRIPTION
**Pull request by awebdeveloper**
### What does this PR do?
Removed info on win8 + bb10 & ubuntu depreciation info

### What testing has been done on this change?
This is just a change to docs

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: www/docs/en/7.x/guide/support/index.md [hunk 1]
```diff
@@ -38,10 +38,9 @@ CLI's shorthand names.
     <tr>
         <th></td>
         <th>android</th>
-        <th>blackberry10</th>
+        <th>blackberry10 (depricated)</th>
```
Comment (shazron) [inline www/docs/en/7.x/guide/support/index.md]: spelling -> correct: "deprecated"

File: www/docs/en/7.x/guide/support/index.md [hunk 2]
```diff
@@ -38,10 +38,9 @@ CLI's shorthand names.
...
         <th>ios</th>
-        <th>Ubuntu</th>
-        <th>wp8<br/>(Windows Phone 8)</th>
+        <th>Ubuntu (depricated)</th>
```
Comment (shazron) [inline www/docs/en/7.x/guide/support/index.md]: spelling -> correct: "deprecated"

Comment (awebdeveloper) [inline www/docs/en/7.x/guide/support/index.md]: done

**Comment by shazron** [comment]
Thanks! Can you copy the file to the `dev` tree as well. This file: `www/docs/en/dev/guide/support/index.md`

**Comment by awebdeveloper** [comment]
done

**Comment by shazron** [comment]
@purplecabbage is the removal of win8 here ok?

**Comment by purplecabbage** [comment]
Nope, it seems there are some mixed up deletions ) from a quick glance.
This is the data that shows up on  https://cordova.apache.org/docs/en/latest/guide/support/index.html right?
Windows column should be renamed to `windows (uwp)` and should still exist.

**Comment by awebdeveloper** [comment]
@purplecabbage  you want me to rename windows column name to windows (uwp) from the existing windows (8.1, 10, Phone 8.1)

**Comment by awebdeveloper** [comment]
@purplecabbage ^^

**Comment by stevengill** [comment]
@awebdeveloper I think that is correct. Rename `windows (8.1, 10, Phone 8.1)` to `windows (uwp)`. 

Also, it looks like you are removing  
```
<td data-col="win8"       class="n"></td>
```
but I think you want to be removing
```
<td data-col="winphone8"  class="n"></td>
```

Also, can you do a rebase on master so this is mergable again?

Thanks!

**Comment by purplecabbage** [comment]
The pr is deleting the wrong column -> winphone8 is the one that is going.
Change the title in the windows column to read windows (uwp)




@purplecabbage
risingj.com

**Comment by janpio** [comment]
I changed a lot about this in https://github.com/apache/cordova-docs/pull/741. I think it would be better to wait for this to get merged and then remove what should be removed (Is there a JIRA issue about that?).

**Comment by awebdeveloper** [comment]
Y not just remove the two columns in that PR itself

**Comment by janpio** [comment]
Because this PR does different things. Also I have no idea why this should happen as I couldn't find an issue that tells the reasoning behind the change with some confirmation from some Cordova person that this is indeed wanted.

**Comment by awebdeveloper** [comment]
Also, following plugins are getting sunsetted

1. file transfer-  https://cordova.apache.org/blog/2017/10/18/from-filetransfer-to-xhr2.html 

Even these are getting sunsetted but i didn't see them in the table on the site
1.device motion -  https://blog.phonegap.com/migrating-from-the-cordova-device-motion-plugin-ddd8176632ed
2. device orientation  - https://blog.phonegap.com/migrating-from-the-cordova-device-orientation-plugin-8442b869e6cc

**Comment by janpio** [comment]
Closing this as the remaining changes are for the wrong column (win8 is actuall the name of the `windows` platform).

**Pull request closed without merging**